packages/tasit-account/package.json

```diff
@@ -34,6 +34,6 @@
     "@babel/preset-env": "^7.1.6",
     "@babel/register": "^7.0.0",
     "chai": "^4.2.0",
-    "mocha": "^5.2.0"
+    "mocha": "^6.0.1"
```

Comment:
@marcelomorgado Any important breaking changes in version 6 that affect us?

Reply:
That works fine here after updating root `package-lock.json` file. I don't know why `greenkeeper` didn't update that. I guess that could be because of `hoist` option from `lerna`.

Reply:
Hmm, that's interesting. Good to know. We can keep an eye on how it handles package-lock.json going forward.

Reply:
Resolving this since mocha 6 is working